src/inc/gcinfo.h

```diff
@@ -34,7 +34,7 @@ const unsigned   this_OFFSET_FLAG  = 0x2;  // the offset is "this"
 // The current GCInfo Version
 //-----------------------------------------------------------------------------
 
-#define GCINFO_VERSION 2
+#define GCINFO_VERSION 3
```

Comment:
If you bump the GCInfo version format you have to also bump the version of R2R format. Is this version bump really necessary since this fix is only for platforms that have not shipped yet?

Reply:
The version bump was not clear to me. I have thought that I need to bump it since there can already be some crossgen-ed 3rd party code that would use the old version. Is that not a problem?

Reply:
> there can already be some crossgen-ed 3rd party

That's correct for x86/x64. This fix is for ARM/ARM64 that do not have this problem.

`GC_INFO_WANTS_REPORT_ONLY_LEAF` is useless flag. It just exists for compatibility with JIT64 convention on .NET Framework x64. I would overload this flag for ARM/ARM64 to track the tailcall presence.